### PR TITLE
api,storage: add index size rpc call for debugging

### PR DESCRIPTION
**Pull request by acud**
add localstore index debugging information through RPC calls

### CODE REVIEW

File: api/inspector.go [hunk 1]
```diff
@@ -125,3 +127,17 @@ func (i *Inspector) PeerStreams() (string, error) {
...
 	return string(v), nil
 }
+
+func (i *Inspector) StorageIndices() (string, error) {
+	data, err := i.ls.DebugIndexCounts()
+	if err != nil {
+		return "", err
+	}
+
+	strBytes, err := json.Marshal(data)
```
Comment (nonsense) [inline api/inspector.go]: I think you can just return the map, and it will automatically get marshalled.

Comment (nonsense) [reply api/inspector.go]: See `DeliveriesPerPeer`.

Comment (acud) [reply api/inspector.go]: will it get marshalled to a json? or as a string?

Comment (nonsense) [reply api/inspector.go]: Depends on the layer above, this is not the layer where we decide if we want to marshal in JSON or another format.

Comment (janos) [reply api/inspector.go]: It would be nicer if we return map here, but that can be changed later if we want to do testing as soon as possible.

**Review by nonsense — APPROVED** [review]

**Review by janos — APPROVED** [review]

**Pull request merged**